source/2019-09-27-the-ember-times-issue-117.md

```diff
@@ -26,80 +32,96 @@ responsive: true
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Accessibility, Compiler Secrets and Modern Ember @ EmberCamp 🐹](#section-url)
+
+
+Last week the Ember community went on another extraordinary trip into the metropolitan wilderness of Illinois:
+[**EmberCamp 2019**](https://embercamp.com) took place in Chicago on September 16th and was insightful, exciting and entertaining as always!
+
+With twelve, versed speakers, the schedule offered thoughtful insights into:
+
+- what's new in Ember with its [first edition, Octane](https://emberjs.com/editions/octane/)
+- how [Embroider](https://github.com/embroider-build/embroider) and Ember will literally build a future together
+- everything you need to know to build **accessible JavaScript apps** in 2019
+- how to level up your app's **performance** with worker super powers or well-balanced animations
+- strategies for stable and reliable test suites
+- ...and many, many more topics!
+
+One particular highlight of the camping trip was the ["May I Ask a Question" live event](https://stackoverflow.blog/2019/09/20/meet-the-developer-who-took-stack-overflow-from-screen-to-stage/)
+that both [Jen Weber (@jwwweber)](https://twitter.com/jwwweber) and [Preston Sego (@nullvoxpopuli)](https://twitter.com/nullvoxpopuli) led and hosted, answering burning questions from the community.
+
+Missed out on the conference, but still want to catch up before the recordings are out?
+Be sure to check out [Jordan Hawker's (@elwayman02)](https://github.com/elwayman02) comprehensive [EmberCamp summary](https://github.com/elwayman02/embercamp), including talk notes and slides for the entire programme.
```

Comment:
Can we link to the Markdown file so that clicking on it shows the summary right away?
```suggestion
Be sure to check out [Jordan Hawker's (@elwayman02)](https://github.com/elwayman02) comprehensive [EmberCamp summary](https://github.com/elwayman02/embercamp/blob/master/2019.md), including talk notes and slides for the entire programme.
```